keyboards/sirius/unigo66/rules.mk

```
@@ -0,0 +1,20 @@
+# MCU name
+MCU = atmega32u4
+
+
+F_CPU ?= 16000000
```

Should just be an `=` here.

```suggestion
F_CPU = 16000000
```